contrib/completion/zsh/_docker

```diff
@@ -2267,6 +2267,7 @@ __docker_swarm_subcommand() {
             _arguments $(__docker_arguments) \
                 $opts_help \
                 "($help)--advertise-addr=[Advertised address]:ip\:port: " \
+                "($help)--datapath-addr=[Data path ip or interface]:ip " \
```

Comment:
I'd make IP uppercase (in "Data path ip...") to be consistent with other descriptions.